common/config/rush/common-versions.json

```diff
@@ -51,12 +51,13 @@
    * This design avoids unnecessary churn in this file.
    */
   "allowedAlternativeVersions": {
-    /**
-     * For example, allow some projects to use an older TypeScript compiler
-     * (in addition to whatever "usual" version is being used by other projects in the repo):
-     */
-    // "typescript": [
-    //   "~2.4.0"
-    // ]
+    "webpack": [
+      // All projects should use this version by default
```

Comment:
You shouldn't need this. As the comment above says, only list the "additional versions" that are allowed.

Reply:
It always complains if I don't put both in, I assume its a bug in how they detect the primary version but I can't see how they choose the primary version (assumed it would be the most common)